Apps/WebClient/src/ClientApp/src/components/report/covid19.vue

```diff
@@ -58,9 +58,9 @@ export default class COVID19ReportComponent extends Vue {
                 b.labResults[0].collectedDateTime
             );
 
-            return firstDate.isAfter(secondDate)
+            return firstDate.isBefore(secondDate)
                 ? 1
-                : firstDate.isBefore(secondDate)
+                : firstDate.isAfter(secondDate)
                 ? -1
                 : 0;
```

Comment:
Nit: are we not dealing with sonar lint issues?

<img width="874" alt="Screen Shot 2021-11-18 at 11 26 38 AM" src="https://user-images.githubusercontent.com/58790456/142483511-6e2c3292-e5dd-4bc1-8b62-0c9200d46d5a.png">